monitoring/monitoring-overview.adoc

```diff
@@ -0,0 +1,34 @@
+:_content-type: ASSEMBLY
+[id="monitoring-overview"]
+= Monitoring overview
+include::modules/common-attributes.adoc[]
+:context: monitoring-overview
+
+toc::[]
+
+[id="about-openshift-container-platform-monitoring"]
+== About {product-title} Container Platform monitoring
+
+{product-title} includes a pre-configured, pre-installed, and self-updating monitoring stack that provides *monitoring for core platform components*. A cluster administrator can xref:../monitoring/configuring-the-monitoring-stack.adoc#configuring-the-monitoring-stack[configure the monitoring stack] with the supported configurations. You also have the option to enable xref:../monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects[monitoring for user-defined projects]. {product-title} delivers monitoring best practices out of the box. A set of alerts are included by default that immediately notify cluster administrators about issues with a cluster. Default dashboards in the {product-title} web console include visual representations of cluster metrics to help you to quickly understand the state of your cluster.
+The {product-title} web console provides xref:../monitoring/accessing-third-party-uis.adoc#accessing-third-party-uis[third-party integrated UIs] such as metrics, alerts, and review of monitoring dashboards. You can access these third-party user interfaces (UIs) either through the web console or the CLI. You can use the **Administrator** or ** Developer** perspective in the {product-title} monitoring dashboard to perform the following tasks:
+
+* xref:../monitoring/managing-metrics.adoc#managing-metrics[Manage metrics]
+* xref:../monitoring/managing-alerts.adoc#managing-alerts[Manage alerts]
+* xref:../monitoring/reviewing-monitoring-dashboards.adoc#reviewing-monitoring-dashboards[Review monitoring dashboards]
+
+// Understanding the monitoring stack
+include::modules/monitoring-understanding-the-monitoring-stack.adoc[leveloffset=+1]
```

Comment:
the name monitoring-understanding-the-monitoring-stack is a bit confusing

Reply:
@lihongyan1 hi! Thank you for your feedback on this. In the past we prepended modules that related to monitoring with "monitoring" so that they could be easily identified.

Reply:
The sentence, "A cluster administrator can xref:../monitoring/configuring-the-monitoring-stack.adoc#configuring-the-monitoring-stack[configure the monitoring stack] with the supported configurations" could come after "A cluster administrator can xref:../monitoring/configuring-the-monitoring-stack.adoc#configuring-the-monitoring-stack[configure the monitoring stack] with the supported configurations." I am suggesting that because the page on configuring the monitoring stack relates to the core OpenShift monitoring components as well as monitoring for user-defined projects.

Reply:
You could remove the bold weighting from "monitoring for core platform components". With the changes in this PR, "monitoring for user-defined projects" is no longer bold, because it is now a link. My view is that having only one of these in bold does not add value when scanning the section.